composer.json

```diff
@@ -50,7 +50,6 @@
     "illuminate/log": "^8.0",
     "illuminate/support": "^8.0",
     "illuminate/view": "^8.0",
-    "laravel-zero/foundation": "^8.0",
```

Comment:
Hi,
Can you explain me why you have chosen to have a copy of foundation in the source code vs using the one provided by laravel-zero?

Reply:
via @QWp6t: 

> Because it autoloads `helpers.php`, which contains a bunch of global functions, one (or more?) of which conflicts with global helper functions provided by WordPress, such as `__()`

Reply:
Please check this composer plugin [https://github.com/mcaskill/composer-plugin-exclude-files](https://github.com/mcaskill/composer-plugin-exclude-files) which was made for this case ( excluding laravel helpers from foundation).
So much better to have Foundation in vendor code..

Reply:
I had considered that, but in my testing, it would require us to instruct users to add extras.exclude-from-files manually to their composer.json file, i.e., we can't just include that with Acorn.

This isn't the end of the world, but it does mean that we'd frequently get issues and bug reports from users who miss this step and can't figure out why there's a "function already defined" error on their site. Additionally, that plugin doesn't run if the user passes `--no-scripts` to `composer install`.

It seemed easier to continue to vendor Foundation for now.

But obviously I'm totally open to the idea of switching back to laravel-zero/foundation if we can come up with an elegant zero-config way of loading it without helpers.php.

Reply:
IMO just add it to composer.json of Sage and you have covered +90% of cases. 
For other uses, like i'm doing adding acorn to the project composer.json, you will need anyway to make some adjustments and use Sage's composer.json as reference to copy useful parts...
for `--no-scripts`, probably you mean ` --no-plugins` which seems an even rarer case..
As further solution could be to create a Roots fork of laravel-zero/Foundation with this little modification to the composer.json ( a github action could automate the process on each release)

Reply:
We're [no longer going to have Acorn as a direct dependency of Sage](https://github.com/roots/sage/pull/2827). That's what prompted me to remove laravel-zero/foundation and re-add Foundation directly into the repo.

We will eventually set up some sort of automated solution, but for right now, I'm focusing my time on getting some better test coverage in Acorn and squashing bugs. When things settle down and I have some time, I'll revisit the idea of setting up our own repo to mirror Foundation with some modifications.